client/src/stories/FloatingButton.stories.js

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react';
+import { FloatingButton } from '../components/FloatingButton';
+
+export default {
+  title: 'Button',
+  component: FloatingButton,
+};
+
+export const AnimatedFloatingButton = () => {
+  const [active, setActive] = useState(false);
```

Comment:
Maybe you can outsource the state into the component file.
With that, you could have a very clean component appeal with only the components after each other. 
Think about a complex page where you have to call a lot of components. Either you have every component and for every component additional lines of code where you set the states OR you put all states into the component itself and when you call the component, you only have 1 line of code, the component itself.

Reply:
it really depends on how the button is used, I think for now it makes the most sense how it is right now